net/peer.go

```diff
@@ -21,10 +21,10 @@ import (
 	"github.com/ipfs/boxo/bitswap/network"
 	"github.com/ipfs/boxo/blockservice"
 	exchange "github.com/ipfs/boxo/exchange"
-	dag "github.com/ipfs/boxo/ipld/merkledag"
+	"github.com/ipfs/boxo/fetcher"
```

Comment:
suggestion: I think it might be worth aliasing this import with a comment explaining why, when I was first reading the consuming code I thought it was working with the `db/fetcher` package - `fetcher` has quite a strong meaning in our codebase.

Same in process.go too.

Reply:
✅